src/interface/web/app/share/chat/layout.tsx

```diff
@@ -20,9 +20,9 @@ export default function RootLayout({
                 httpEquiv="Content-Security-Policy"
                 content="default-src 'self' https://assets.khoj.dev;
                        script-src 'self' https://assets.khoj.dev 'unsafe-inline' 'unsafe-eval';
-                       connect-src 'self' https://ipapi.co/json ws://localhost:42110;
+                       connect-src 'self' data: blob: https://ipapi.co/json ws://localhost:42110;
```

Comment:
Why are `data:` and `blob:` required for `connect-src` in CSP? 

Note: similar changes are done to the CSP in `src/interface/web/app/layout.tsx` and `src/interface/web/app/chat/layout.tsx`